website/docs/r/event_orchestration_service.html.markdown

```diff
@@ -0,0 +1,215 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_service"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-service"
+description: |-
+  Creates and manages a Service Orchestration for a Service.
+---
+
+# pagerduty_event_orchestration_service
+
+A [Service Orchestration](https://support.pagerduty.com/docs/event-orchestration#service-orchestrations) allows you to create a set of Event Rules. The Service Orchestration evaluates Events sent to this Service against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Service Orchestration for further processing.
+
+**Note:** If you have a Service that uses [Service Event Rules](https://support.pagerduty.com/docs/rulesets#service-event-rules), you can switch to [Service Orchestrations](https://support.pagerduty.com/docs/event-orchestration#service-orchestrations) at any time. Please read the [Switch to Service Orchestrations](https://support.pagerduty.com/docs/event-orchestration#switch-to-service-orchestrations) instructions for more information.
+
+## Example of configuring a Service Orchestration
+
+This example shows creating `Team`, `User`, `Escalation Policy`, and `Service` resources followed by creating a Service Orchestration to handle Events sent to that Service.
+
+This example also shows using `priority` [data source](https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/data-sources/priority) to configure `priority` action for a rule. If the Event matches the first rule in set "step-two" the resulting incident will have the Priority `P1`.
+
+This example shows a Service Orchestration that has nested sets: a rule in the "start" set has a `route_to` action pointing at the "step-two" set.
+
+The `catch_all` actions will be applied if an Event reaches the end of any set without matching any rules in that set. In this example the `catch_all` doesn't have any `actions` so it'll leave events as-is.
+
+
+```hcl
+resource "pagerduty_team" "engineering" {
+  name = "Engineering"
+}
+
+resource "pagerduty_user" "example" {
+  name  = "Earline Greenholt"
+  email = "125.greenholt.earline@graham.name"
+  teams = [pagerduty_team.engineering.id]
+}
+
+resource "pagerduty_escalation_policy" "foo" {
+  name      = "Engineering Escalation Policy"
+  num_loops = 2
+
+  rule {
+    escalation_delay_in_minutes = 10
+    target {
+      type = "user"
+      id   = pagerduty_user.example.id
+    }
+  }
+}
+
+resource "pagerduty_service" "example" {
+  name                    = "My Web App"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.example.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+data "pagerduty_priority" "p1" {
+  name = "P1"
+}
```

Comment:
**Kudos:** Yay!! Makes this example super realistic _and_ copy/pastable for people wanting to learn more 🎉